### ISSUE TITLE
Improve README.md

### ISSUE THREAD

**Issue by tobiasmuecksch** [s-a/time-track#5]
The readme file is a little unclear on how to use time-track. Without the gif-image I would not have understood that you start tracking the time by simply typing `tt` and stopping likewise.

In my opinion there should be two new sections in the read me. One that describes the workflow and one that describes how to develop (for example add `node lib/cli.js -s test` or `npm install -g` for testing purposes).

**Comment by s-a** [comment]
I agree. We are also missing a licence file :smile_cat: I will start to work an that this weekend.
